src/db/wallets/walletNonce.ts

```diff
@@ -58,8 +102,16 @@ export const recycleNonce = async (
   walletAddress: Address,
   nonce: number,
 ) => {
+  if (isNaN(nonce)) {
```

Comment:
I've sometimes seen NaN nonces in the recycled nonces list too. Were you able to find out why? Is this log getting printed?

Reply:
I saw it too